Sources/ComposableArchitecture/Documentation.docc/Tutorials/BuildingSyncUps/07-SyncUpDetailNavigation/SyncUpDetailNavigation-01-code-0006.swift

```diff
@@ -2,8 +2,8 @@ import ComposableArchitecture
 import SwiftUI
 
 @Reducer
-struct App {
-  @Reducer
+struct AppFeature {
+  @Reducer(state: .equatable)
```

Comment:
Necessary for compilation, and I added a note on this in the tutorial where this gets added.